frontend/javascripts/libs/utils.ts

```diff
@@ -886,7 +886,7 @@ export function castForArrayType(uncastNumber: number, data: TypedArray): number
 }
 
 export function convertNumberTo64Bit(num: number | null): [Vector4, Vector4] {
-  if (num == null) {
+  if (num == null || isNaN(num)) {
```

Comment:
not sure how I triggered this, but an error was thrown here at one point which is why I slipped this fix in here.